### ISSUE TITLE
Please add a Documentation entry to the optuna.org header nav bar

### ISSUE THREAD

**Issue by bionicles** [optuna/optuna#4725]
### Motivation

Right now, on optuna.org, there's no direct link to the docs at the top of your screen. This would save clicking and help devs find the docs without hunting around for the link.

### Description

Simply add another header nav bar list item with a link to https://optuna.readthedocs.io/en/stable/index.html

### Alternatives (optional)

The user can click "installation" and find docs on that page, but this forces extra hunting around, someone who just wants docs and has already installed optuna might prefer to jump directly to the docs

### Additional context (optional)

![image](https://github.com/optuna/optuna/assets/24532336/666ebd6f-eda0-4591-9fa6-485ec90a99d5)

**Comment by contramundum53** [comment]
The docs link is here :)

<img width="1169" alt="image" src="https://github.com/optuna/optuna/assets/24619922/6faa8d4c-2977-4ad7-bdca-3a4aa862d907">

**Comment by toshihikoyanase** [comment]
Thank you for your request.

At present, our header navigation bar is intended for in-page navigation only, and it does not include any external links. As you mentioned, we have positioned links to the documentation in the middle or the bottom of the page, as follows:

<img width="848" alt="image" src="https://github.com/optuna/optuna/assets/3255979/8cdcaed6-3f14-4da5-8bf7-184313cf69be">

On the other hand, I recognized that the accessibility of the documentation is indeed a concern. It's clear that we need to refine our site design to ensure users can locate the links with ease. I appreciate your highlighting this issue.
